src/demo-app/datepicker/datepicker-demo.html

```diff
@@ -1,6 +1,25 @@
 <h1>Work in progress, not ready for use.</h1>
 
-<input [mdDatepicker]="dp" [(ngModel)]="date">
-<button (click)="dp.openStandardUi()">open</button>
-<button (click)="dp.openTouchUi()">open (mobile)</button>
-<md-datepicker #dp></md-datepicker>
+<p>
+  <md-checkbox [(ngModel)]="touch">Use touch UI</md-checkbox>
+</p>
+<br>
+<p>
+  <input [mdDatepicker]="dp" [(ngModel)]="date">
+  <md-datepicker-trigger [for]="dp"></md-datepicker-trigger>
+  <md-datepicker #dp [touchUi]="touch"></md-datepicker>
+</p>
+<p>
+  <md-input-container>
+    <input mdInput [mdDatepicker]="dp2" [(ngModel)]="date">
+    <md-datepicker-trigger mdSuffix [for]="dp2"></md-datepicker-trigger>
```

Comment:
How about making the datepicker-trigger a button? I think part of the weirdness for me is that the input should itself be a trigger, so adding something else called "trigger" threw me off.  

Jeremy and I were thinking maybe: 

```html
<input [mdDatepicker]="dp">
<button mdDatepickerToggle></button>
<md-datepicker #dp></md-datepicker>
```